grakn-test-tools/pom.xml

```diff
@@ -69,7 +69,12 @@
             <artifactId>embedded-redis</artifactId>
             <version>${redis-test.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>com.github.zxl0714</groupId>
+            <artifactId>redis-mock</artifactId>
```

Comment:
I just had a deeper look at this library. I'm a bit conflicted. On one side it does what a minimal in memory redis mock should do. Opens a connection on the port, reads commands and mocks redis as a Map. So you can read and write from it and it also supports TTL of data.
On the other side it's full of typos and the code is not amazing. 

In the next set of changes maybe I can scan through where we use Redis in tests and see if I can use this instead for stricter unit tests where we are not relying too much on testing Redis behaviour.